tls/s2n_config.c

```diff
@@ -811,7 +802,9 @@ int s2n_config_add_ticket_crypto_key(struct s2n_config *config,
         notnull_check(config->ticket_key_hashes = s2n_array_new(SHA_DIGEST_LENGTH));
     }
 
-    GUARD(s2n_verify_unique_ticket_key(config, hash_output, &insert_index));
+    if (s2n_verify_unique_ticket_key(config, hash_output, &insert_index) < 0) {
+        return 1;
```

Comment:
Why 1 instead of GUARD?

Reply:
I would like to identify this from the rest. I am logging an error when s2n_config_add_ticket_crypto_key returns -1. Didn't want to do that when a key has already been added before. Wanted to log it at debug level.

Reply:
Will returning error of S2N_ERR_T_USAGE type help here? Then you can use debug level on that one, but error-level on everything else.

Reply:
Updated it to return error of S2N_ERR_T_USAGE type. I was thinking I'll do something like below 

```
        char test[] = "Exact expected error message";
        if (strstr(s2n_strerror_debug(s2n_errno, "EN"),  test) == NULL) {
              // log at debug level and error for rest
        }
```